components/core/Base/AntInputBoolComponentBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace AntDesign
+{
+    public abstract class AntInputBoolComponentBase : AntInputComponentBase<bool>
+    {
+        [Parameter] public bool AutoFocus { get; set; }
+
+        private bool _checked;
+        [Parameter]
+        public bool Checked
+        {
+            get { return _checked; }
+            set
+            {
+                _checked = value;
+                if (_checked != Value)
+                {
+                    Value = _checked;
+                }
+            }
+        }
+
+        [Parameter]
+        public EventCallback<bool> OnChange { get; set; }
+
+        /// <summary>
+        /// Gets or sets a callback that updates the bound checked value.
+        /// </summary>
+        [Parameter]
+        public virtual EventCallback<bool> CheckedChanged { get; set; }
+
+        [Parameter]
+        public bool Disabled { get; set; }
+
+        internal virtual string PrefixCls { get; }
+
+        protected virtual void SetClass()
+        {
+            ClassMapper.Clear()
+                .Add(PrefixCls)
+                .If($"{PrefixCls}-rtl", () => RTL);
+        }
```

Comment:
I think it's no need to set class in base class.

Reply:
Do you want me to move `PrefixCls` out of base class as well?

Reply:
Yes, it doesn't seem to make much sense to separate it from the specific component.